src/osd/scrubber/pg_scrubber.cc

```diff
@@ -19,6 +19,7 @@
 #include "messages/MOSDScrubReserve.h"
 #include "osd/OSD.h"
 #include "osd/PG.h"
+#include "include/object_fmt.h"
```

Comment:
not needed, as already included thru osd/osd_types_fmt.h

Reply:
sure. dropped.